client/changelist/change.go

```diff
@@ -38,8 +38,10 @@ type TufChange struct {
 // TufRootData represents a modification of the keys associated
 // with a role that appears in the root.json
 type TufRootData struct {
-	Keys     data.KeyList `json:"keys"`
-	RoleName string       `json:"role"`
+	ReplaceKeys data.KeyList `json:"keys"`
```

Comment:
We've never done anything with root keys up to now so that actions can be treated slighty differently when dealing with that role specifically and it won't break existing clients.